### PR TITLE
feat(balance): Reduce Supercapacitor price from 12 per energy down to 2 per energy to be more inline with other batteries that are in the 1.7 to 1.5 per energy range.

### PR DESCRIPTION
**Pull request by Zitchas**
**Balance:**

## Summary
Reduced the cost of the supercapacitor from 9000 to 1500

## Reasoning
LP576a Battery Pack: 1.74 per unit of energy, 575/ton
LP288a Battery Pack: 1.73 per unit of energy, 520/ton
LP144a Battery Pack: 1.66 per unit of energy, 480/ton
LP072a Battery Pack: 1.59 per unit of energy, 440/ton
LP036a Battery Pack: 1.50 per unit of energy, 400/ton
Supercapacitor: 12.00 per unit of energy, 375/ton

As one proceeds from the biggest human battery down to the smallest battery, each step becomes both less dense and less expensive. Starting at 1.739 credits per unit of energy and a density of 575/ton, and decreasing down to the smallest battery at 1.50 credits per unit of energy and a density of 400/ton. Then we get the Supercapacitor, that has less density than even the smallest and least dense of the batteries, and yet is priced at 12.00 credits per unit of energy. Beyond that, in terms of final price, the supercapacitor is 1.5x *higher* than the LP036a battery. Even if one assumes that it's absolutely small size and ability to be tucked into smaller spaces is a noteworthy advantage, it's performance really doesn't merit such a massive increase in price.

As such, this PR reduces its price down to 1500, which is 2.00 credits per energy capacity. I picked this because it thus still retains a bit of a "convenience premium" for being so small, while also allowing it to be used as a way to skimp for people (such as new pilots) who really want to live close to the line and downgrade their energy as far as possible to pay off as much as they can right off the start.

## Impact

Two main impacts: 

This makes supercapacitors actually useful for people who want to swap out their LP036a for something smaller to save some money, which makes them actually useful for something other than just being smaller.

The biggest impact is in regards to fighter affordability. The Lance cost 357900 to purchase. Of that cost, 36000 is for the supercapacitors alone. Reducing the supercapacitor price down to 1500 correspondingly drops the price of the Lance to 327900. This is a 9% reduction in shipyard purchase price. It isn't going to make or break buying the Lance by itself, but it definitely makes it a little less painful to buy and lose.

## Alternate
If one wanted to actually follow the battery progression, then the supercapacitor should actually be reduced down to something closer to 1050, which would be 1.40 credits per energy; or perhaps 1100 which would be 1.46 credits per energy. Both are acceptable to me, depending on how much value "cram 750 energy into a random 2 ton space someone has left over" actually is. I'd argue that it isn't particularly valuable, personally, and would even prefer these lower values; however I went with the higher 2.00 per energy and a 1500 price tag in order to err on the conservative side and not risk dropping it too far.

As such, I'd consider the price in this PR to be a reasonable one, but if people think that 750 energy really isn't that valuable or useful, I'd be happy to drop it down to 1000 (1.33 credits per energy) or either of the other values listed above. Realistically, changes to the price at this point really only affect fighter purchases or brand new pilots, so lower is better but there's very limited gains to be had by going much lower.

### CODE REVIEW

**Review by Quantumshark — APPROVED** [review]
Seems reasonable. I imagine fighters are the primary use case for supercapacitors, so the main effect of this should be to make replacing fighters less expensive. It'll also make looting supercapacitors less profitable, which is also a sensible change, I feel.

**Comment by Zitchas** [comment]
Yes, it would make looting a bit less profitable as well. I hadn't thought to include that as a benefit, but I suppose it would likely be one too.

**Comment by Hecter94** [comment]
I was always under the impression that the price difference between a supercapacitor and a battery was more to do with lore than balance. 
Representing that the two are fundamentally different technologies, and the cost of creating what is essentially an oversized capacitor is far higher than making a small, standard lithium polymer battery. 
Effectively, hyper-miniaturization comes with a significant cost that a larger outfit won't have. 

If that lore point isn't a major concern, however, then from a balance standpoint, I find this change to be a reasonable one. 

<details>
<summary>Mapping out cost vs energy capacity</summary>

image

</details>

From this, we can see a couple of outliers, 

- Hai Chasm Battery
- Supercapacitor
- Small Battery Module
- Ka'het Reserve Accumulator
- Solar Battery 
- Crystal Capacitor

Of those, the Small Battery Module, Solar Battery, and Ka'het Reserve Accumulator are all technology from higher tier races, making sense to be outliers in terms of cost per performance. 
Crystal capacitor is a standout in terms of performance, boasting the second highest capacity/space, only behind the Nanotech Battery, so in terms of a "performance cost," I don't think it's unreasonable for it to be an outlier either.
The outliers that need to be examined are the Hai Chasm Battery and Supercapacitor. 

Looking at the supercapacitor, I prefer the $1,050 price point, which is the closest to being on curve when considering other human batteries. 

| name                | Cost/Space | Increase |
|---------------------|------------|----------|
| LP576a Battery Pack |       1.74 | 0.01         |
| LP288a Battery Pack |       1.73 | 0.06      |
| LP144a Battery Pack |       1.67 | 0.08       |
| LP072a Battery Pack |       1.59 | 0.09      |
| LP036a Battery Pack |       1.50 | 0.10       |
| Supercapacitor      |       1.40 |     |

However, if we want to preserve the lore point I mentioned above, I suggest $1,400. This keeps it more expensive per unit of energy than regular batteries while still being less than a quarter of the cost.
| name                | Cost/Space | Increase |
|---------------------|------------|----------|
| Supercapacitor      |       1.87 | 0.13     |
| LP576a Battery Pack |       1.74 | 0.01     |
| LP288a Battery Pack |       1.73 | 0.06     |
| LP144a Battery Pack |       1.67 | 0.08     |
| LP072a Battery Pack |       1.59 | 0.09     |
| LP036a Battery Pack |       1.50 |          |

tl;dr: I like this change overall; only one small lore point to note. I prefer $1,050 to be on curve or $1,400 if we want to maintain that lore point, and we might want to look at the Hai Chasm battery in another PR.

**Comment by Amazinite** [comment]
1500 vs 1400 is so negligible that I think keeping it at 1500 is fine.

**Comment by Zitchas** [comment]
I was going to offer to change it to 1050, or ask if 1000 would be alright, as even numbers look better; but this works. :)

From the lore perspective, maintaining it as an outlier doesn't feel like it is serving anything. If this was some middle of the pack or otherwise brought something really distinctive to the game, then maintaining a very sharply unusual price would be really justified. But as is, being statistically just the smallest battery exactly in the battery density curve... It's too important as the only fighter-caliber battery to handicap it with a price that just punishes fighter usage, which is already a problem.
